specs/protocol/differences-from-optimism.md

```diff
@@ -71,7 +72,17 @@ The followings are components that are used to run different types of nodes:
 **NOTE:** Here `L2 EL client` means `kroma-geth` and `L2 CL client` means `kroma-node`. `L2 EL client` can
 be expanded to other clients for pragmatic decentralization.
 
-### Adding field to System Configuration
+## Validator
+
+### ZK fault proof
+
+Instead of [cannon], Kroma uses zkVM for [ZK fault proof][g-zk-fault-proof].
+
+[cannon]: https://github.com/ethereum-optimism/cannon
```

Comment:
```suggestion
### ZK fault proof system

Instead of fault proof system based on [cannon], Kroma uses zkVM for [ZK fault proof][g-zk-fault-proof].

[cannon]: https://github.com/ethereum-optimism/cannon

### PoS Validator System

Kroma has its own [Pos validator system](validator-v2/overview.md) utilizing its governance token.
```

Reply:
I'll go with DPoS instead of PoS, thanks!